gustaf/io/meshio.py

```diff
@@ -18,58 +19,96 @@
     # from meshio import Mesh as MeshioMesh
 
 
-def load(fname):
+def load(fname, set_boundary=False, return_only_one_mesh=True):
```

Comment:
```suggestion
def load(fname, set_boundary=True):
```
I think this is a nicer default